src/Client/index.html

```diff
@@ -6,7 +6,7 @@
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
 		<link rel="stylesheet" href="css/site.css">
-		<link rel="shortcut icon" type="image/png" href="/Images/safe_favicon.png"/>
+		<link rel="shortcut icon" type="image/png" href="/images/safe_favicon.png"/>
```

Comment:
this wil break the image

Reply:
The images folder is in lower case, I'm afraid upper case won't work in case-sensitive systems, like linux.

Reply:
https://github.com/SAFE-Stack/SAFE-BookStore/blob/master/build.fsx#L267

Reply:
Got, I guess we should either keep all images folder name as capitalized or lower-cased, I think I hit this problem when I run SAFE-BookStore locally on a Linux laptop.